framework/auth/__init__.py

```diff
@@ -48,6 +49,7 @@ def logout():
             del session.data[key]
         except KeyError:
             pass
+    Session.remove(Q('_id', 'eq', session._id))
```

Comment:
Makes sense. Yeah, the idea with this code review was to try to find all these little issues. Alex wouldn't know what they might be, nor would I, so I'm hoping between you and @sloria we can uncover whatever might be a related problem.